### PR TITLE
SCF-62: Standardize buttons

### PR DESCRIPTION
**Pull request by Gabe-Mitnick**
[SCF-62](https://scfrontend.atlassian.net/browse/SCF-62?atlOrigin=eyJpIjoiOTM1ODQ2MzczYzdmNGZlY2EwODIyYjljMmIyNGVhNGMiLCJwIjoiaiJ9) involved DRYing the styles of the buttons around the site. Now, all the aspects of button style that are the same across different buttons live in index.scss. index.scss and Navbar.scss are both newly sassified. This PR will change how some buttons look, primarily by making corner radii all be 5px in concordance with the designs. Github says there's a merge conflict so I'll figure that out later.

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -35,6 +34,81 @@ code {
...
+  color: #2b2b2b;
+  background-color: #c4c4c4;
+  &:hover {
+    background-color: #747474;
+  }
+}
+
+
+.save-button {
+  margin-bottom: 20px;
```
Comment (tankaren) [inline src/index.scss:93]: i have a feeling we actually want to remove the margin-bottom: 20px from buttons, but we can test that more

File: src/index.scss [hunk 2]
```diff
@@ -35,6 +34,81 @@ code {
...
+  float: right;
+  padding: 11px 26px;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.cancel-button {
+  margin-bottom: 20px;
```
Comment (tankaren) [inline src/index.scss:101]: same as above comment on save-button

Comment (Gabe-Mitnick) [reply src/index.scss:101]: First example of these buttons that i found just now is in the gallery image upload, and when I remove those margin-bottoms, it makes it look bad. I can't find the edit button on the admin club page to check there, but I imagine if we took away the margin from the buttons, we would need to add it as padding or margin on every container that holds these buttons. Do you want me to try to do that? I'm hoping to merge this PR ASAP before it starts accumulating merge conflicts.

Comment (tankaren) [reply src/index.scss:101]: Yea good point. For now, I'll just make a ticket that we should come back to this later.

**Review by tankaren — APPROVED** [review]

**Pull request merged**